cohesity/src/info

```diff
@@ -1,18 +1,18 @@
 {'author': 'Sven Rueß, sritd.de, Bastian Kuhn, bastian-kuhn.de',
- 'description': 'Cohesity checks',
+ 'description': 'Cohesity checks\n',
```

Comment:
Don't understand the reason for the line break here? Is it cause of CMK 2.0?

Reply:
Yes, This is done by Checkmk directly. I configured it in GUI.